boilerplate/fullstack/astro/src/config/appInfo.ts

```diff
@@ -3,6 +3,6 @@ export const appInfo = {
     appName: "SuperTokens Astro demo app",
     apiDomain: "http://localhost:4321",
     websiteDomain: "http://localhost:4321",
```

Comment:
what about the issue one person was saying that choosing emailpassword + social login only makes an app with social login?

Reply:
![image](https://github.com/user-attachments/assets/96be681c-9748-4636-97d2-10723a290eda)

Seems to work fine.

Reply:
Tried it with multiple combinations (just social, just email/pass, social + paswordless, social + email/pass, all three at once) too.